webpack/assets/javascripts/foreman_dashboard.js

```diff
@@ -0,0 +1,138 @@
+import $ from 'jquery';
+import { doesDocumentHasFocus } from './react_app/common/document';
+import { notify } from './foreman_toast_notifications';
+import { activateTooltips } from './foreman_tools';
+import { translate as __ } from './react_app/common/I18n';
+
+$(document).on('ContentLoad', () => {
+  if (window.foreman_url(window.location.pathname) === '/') {
```

Comment:
I think this needs to be the other way around:
```suggestion
  if (window.foreman_url('/') === window.location.pathname) {
```